src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visits/VisitService.kt

```diff
@@ -196,7 +196,6 @@ class VisitService(
         startDateTime = updateVisitRequest.startDateTime,
         endDateTime = endDateTime,
         location = visit.location,
-        roomDescription = updateVisitRequest.room,
```

Comment:
Did you mean for these to go back ?

Reply:
ah yes, i meant to mention these are intellij warnings about unused params etc